### ISSUE TITLE
Decision Tree Classifier 决策树

### ISSUE THREAD

**Issue by xiaoma229** [xiaoma229/blogger#6]
import pandas as pd
from sklearn.model_selection import train_test_split
from sklearn.tree import DecisionTreeClassifier
from sklearn import tree
music_data = pd.read_csv('music.csv')
X = music_data.drop(columns = ['genre'])
Y = music_data['genre']
model = DecisionTreeClassifier()
model.fit(X, Y)

tree.export_graphviz(model, out_file = 'music-recommender.dot', 
                    feature_names = ['age', 'gender'], 
                    class_names = sorted(Y.unique()), 
                    label = 'all', 
                    rounded = True, 
                    filled = True)
